core/src/mindustry/editor/MapEditorDialog.java

```diff
@@ -252,28 +252,29 @@ private void playtest(){
             state.rules = Gamemode.editor.apply(lastSavedRules.copy());
             state.rules.sector = null;
             state.map = new Map(StringMap.of(
-                "name", "Editor Playtesting",
-                "width", editor.width(),
-                "height", editor.height()
+            "name", "Editor Playtesting",
+            "width", editor.width(),
+            "height", editor.height()
```

Comment:
Unnecessary changes to indentation.

Reply:
most unnesesary formatting changes were performed by ide. i made it use recomended formatting configuration for mindustry, so i assumed any changes would not interfer, i can of corse fix it.